README.md

```diff
@@ -74,6 +86,7 @@ Then, copy the following two Logtail appenders to your `classpath:/logback.xml`
     <appender name="LogtailHttp" class="com.logtail.logback.LogtailAppender">
         <appName>MyApp</appName>
         <ingestKey>${LOGTAIL_INGEST_KEY}</ingestKey>
+        <batchSize>10</bathSize>
```

Comment:
Looks like a typo. Also any reason for this change? Or was this just for testing?

Reply:
Since the readme.md file uses a sample configuration, I copied and pasted the one I used.
At the time I added that line, no problem to remove it if deemed necessary.

Reply:
The default for `batchSize` is 1000 when unspecified, and honestly I would rather keep that. Could you revert this line if you don't mind?